text/stacks/0003-stack-descriptor.md

```diff
@@ -0,0 +1,88 @@
+# Stack Descriptor and Tooling
+
+## Summary
+
+The stack-building codebase,
+[`create-stack`](https://github.com/paketo-buildpacks/stacks/tree/main/create-stack)
+should be moved into the `jam` tooling CLI, and allow users to configure their
+stack build via a common configuration file.
+
+## Motivation
+
+Today, `create-stack` is highly coupled to the `bionic` stacks we currently
+support. This makes it difficult to support alternative stacks and platforms or
+to allow the codebase to be reused by external parties for developing their own
+stack.
+
+## Detailed Explanation
+
+The `create-stack` codebase will be refactored to allow users to build stacks
+using a configuration file with the following format:
+
+```toml
+id = "<stack id>"                # required
+homepage = "<homepage uri>"      # optional
+maintainer = "<maintainer name>" # optional
+
+platforms = ["linux/amd64", "linux/arm64", "<other-platform>"] # optional; default = ["linux/amd64"]; choices defined in https://docs.docker.com/desktop/multi-arch/
+
+[distro]
+  name = "<os distro name>"       # optional
+  version = "os distro version>"  # optional
```

Comment:
Should we be using values (`ID` and `VERSION_ID`) in `/etc/os-release` instead of user-provided values?

Or we could make these optional and fail if they do not match `/etc/os-release`.

Reply:
It makes sense to just use what exists in `/etc/os-release`. We can probably omit these labels otherwise since the labels themselves are not required by the spec. I added some details about this in the `Stack Spec Requirements` section.